extensions/wikia/FounderEmails/maintenance.php

```diff
@@ -39,6 +41,16 @@
 $wikiId = null;
 if ( isset( $options['wikiId'] ) && is_numeric( $options['wikiId'] ) ) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "wikiId" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)